src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
+   * @typedef {object} PersonObject
```

Comment:
Shouldn't this also be `Object`?
```suggestion
   * @typedef {Object} PersonObject
```

Reply:
Ugh, I keep getting `Object` and `string` capitalization backwards! Good find